sap/cli/dataelement.py

```diff
@@ -98,12 +99,24 @@ def define(connection, args):
     dataelement.normalize()
 
     validation_issue_key = dataelement.validate()
-    if validation_issue_key == VALIDATION_ISSUE_KEYS['domain_name_not_defined']:
-        console.printerr('Domain name must be provided (--domain_name) if the type (--type) is "domain"')
-        return
-    if validation_issue_key == VALIDATION_ISSUE_KEYS['data_type_not_defined']:
-        console.printerr('Data type name must be provided (--data_type) if the type (--type) is "predefinedAbapType"')
-        return
+
+    match validation_issue_key:
+        case DataElementValidationIssues.MISSING_DOMAIN_NAME:
+            console.printerr(
+                'Domain name must be provided (--domain_name) if the type (--type) is "domain"')
+
+        case DataElementValidationIssues.MISSING_TYPE_NAME:
+            console.printerr(
+                'Data type name must be provided (--data_type) if the type (--type) is "predefinedAbapType"')
+
+        case DataElementValidationIssues.NO_ISSUE:
+            pass
+
+        case _:
+            raise SAPCliError("BUG: please report a forgotten case DataElementValidationIssues")
```

Comment:
Tested but the force pushed commit is not presetn yet: https://github.com/buca92/sapcli/commit/7ca8567f4b1a35eb91ec4853e711a24d7b699919#diff-01319a9d5bb09fa62975ace1e4391b81e6d263af504892bae6dcd4463fdf846fR497

Reply:
It is already there ;). Even the Coverage Report shows 100 % again